static/app/views/settings/organizationMembers/inviteBanner.spec.tsx

```diff
@@ -0,0 +1,173 @@
+import {browserHistory} from 'react-router';
+import moment from 'moment';
+
+import {act, render, screen} from 'sentry-test/reactTestingLibrary';
+
+import {MissingMember} from 'sentry/types';
+import {DEFAULT_SNOOZE_PROMPT_DAYS} from 'sentry/utils/promptIsDismissed';
+import {InviteBanner} from 'sentry/views/settings/organizationMembers/inviteBanner';
+
+const missingMembers = {
+  integration: 'github',
+  users: TestStubs.MissingMembers() as MissingMember[],
+};
+
+const noMissingMembers = {
+  integration: 'github',
+  users: [] as MissingMember[],
+};
+
+describe('inviteBanner', function () {
+  beforeEach(function () {
+    MockApiClient.clearMockResponses();
+    MockApiClient.addMockResponse({
+      url: '/organizations/org-slug/missing-members/',
+      method: 'GET',
+      body: [],
+    });
+    MockApiClient.addMockResponse({
+      url: '/prompts-activity/',
+      method: 'GET',
+      body: {
+        dismissed_ts: undefined,
+        snoozed_ts: undefined,
+      },
+    });
+    (browserHistory.push as jest.Mock).mockReset();
```

Comment:
```suggestion
    jest.mocked(browserHistory.push).mockReset();
```

Reply:
This is what you usually want instead of casting.

Reply:
i actually don't need this !